app/src/main/java/org/mozilla/focus/settings/ExperimentsSettingsFragment.kt

```diff
@@ -37,6 +51,7 @@ class ExperimentsSettingsFragment : PreferenceFragmentCompat(), SharedPreference
         super.onPause()
         preferenceScreen?.sharedPreferences?.unregisterOnSharedPreferenceChangeListener(this)
```

Comment:
Do we still need the pref handling code in here? determineEngine() only seems to use the override?

Reply:
Oh, right. This is no longer needed.